JSDemos/Demos/List/ListEditingAndAPI/description.md

```diff
@@ -1 +1,23 @@
-The **List** is&nbsp;a&nbsp;UI component that represents a&nbsp;collection of&nbsp;items in&nbsp;a&nbsp;scrollable list. This demo displays different delete modes and shows how you can handle the **selectionChanged** event.
\ No newline at end of file
+The [List](/Documentation/ApiReference/UI_Components/dxList/) displays a collection of items as a scrollable list. To create a list, pass an array of items to the [dataSource](/Documentation/ApiReference/UI_Components/dxList/Configuration/#dataSource) property. 
+
+To allow users to delete items, set the [allowItemDeleting](/Documentation/ApiReference/UI_Components/dxList/Configuration/#allowItemDeleting) property to **true**. Use the "Allow deletion" checkbox under the List to see how this property works.
+
+Use the "Deletion UI type" drop-down menu under the List widget to try different deletion behaviors described below. Menu items correspond to [itemDeleteMode](/Documentation/ApiReference/UI_Components/dxList/Configuration/#itemDeleteMode) property values. 
+    
+- *"static"* (default for desktops)    
+Click an X button next to a list item to delete it. 
+
+- *"toggle"*    
+Click a **minus** icon next to a list item to display the Delete button. 
+
+- *"slideButton"*    
+Swipe left or right to show the Delete button next to the list item. The content stays in place.  
```

Comment:
"The content stays in place" Do you mean the content does not move? As in, the content is not affected?

Reply:
Yes. In some cases, list items are moved off-canvas but in this case, it doesn't move

Reply:
```suggestion
Swipe left or right to show the Delete button next to the list item. The content of the list item does not move.  
```